libs/elasticsearch/tests/integration_tests/_async/test_vectorstores.py

```diff
@@ -792,7 +822,20 @@ async def test_elasticsearch_with_relevance_score(
         output = await docsearch.asimilarity_search_by_vector_with_relevance_scores(
             embedding=embedded_query, k=1
         )
-        assert output == [(Document(page_content="foo", metadata={"page": "0"}), 1.0)]
+        doc, score = output[0]
+        assert doc == Document(page_content="foo", metadata={"page": "0"})
+        # Use the client to pick a tolernace for based on ES version
+        info = await docsearch.client.info()
+        es_version = info["version"]["number"]
+        major, minor = map(int, es_version.split(".")[:2])
+        if (major, minor) >= (8, 14):
+            # if ES 8.14+ then relax the assertion to a tolerance delta
```

Comment:
Same for this one. It looks like 8.15 is the oldest version that would not require version checks, right?